docs/4.2/quickstart.md

```diff
@@ -164,10 +163,10 @@ accessible IP.
 
 !!! warning "Warning":
     For the purposes of this quickstart we are using the
-    `--insecure` flag which allows us to skip configuring the HTTP/TLS
+    `--insecure-no-tls` flag which allows us to skip configuring the HTTP/TLS
```

Comment:
These should actually both be set to `--insecure` - that'll be correct. Sorry for the confusion.

Reply:
revised